packages/terra-clinical-result/src/proptypes/valuePropTypes.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+
+const valueQuantityPropShape = PropTypes.shape({
+  /**
+   *  Numeric Value for an Observation Result
+   */
+  value: PropTypes.number,
+  /**
+   *  Unit of Measure representation for an Observation Result
+   */
+  unit: PropTypes.string,
+  /**
+   *  Optional (FHIR) - System that defines coded unit form
+   */
+  system: PropTypes.string,
+  /**
+   *  Optional (FHIR) - Coded form of the unit
+   */
+  code: PropTypes.string,
+});
+
+const valueStringPropShape = PropTypes.shape({
+  /**
+   *  String Value for an Observation Result
+   */
+  value: PropTypes.string,
+});
+
+const valueNullPropShape = PropTypes.shape({
+  /**
+   *  Using Object to set Null as Value for an Observation Result
+   */
+  value: PropTypes.object,
+});
+
+export default {
+  valueQuantityPropShape,
+  valueStringPropShape,
+  valueNullPropShape,
+};
+export {
```

Comment:
Do we need both exports?

Reply:
removed extraneous export https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-9f4b706335e0ffe997f506325c438202L36